github/github.go

```diff
@@ -880,6 +880,32 @@ func cloneRequest(r *http.Request) *http.Request {
 	return r2
 }
 
+// formatRateLimitResetDuration formats d to look like "[rate reset in 2s]" or
+// "[rate reset in 87m02s]" for the positive durations. And like "[rate limit was reset 87m02s ago]"
+// for the negative cases.
+func formatRateReset(d time.Duration) string {
+	isNegative := d < 0
+	if isNegative {
+		d *= -1
+	}
+	secondsTotal := int(d.Seconds())
+	minutes := int((secondsTotal) / 60)
```

Comment:
Yep, this is much better.

[`unconvert`](https://github.com/mdempsky/unconvert) tool by @mdempsky helps finds opportunities like this.